contrib/IECoreUSD/src/IECoreUSD/PointInstancerAlgo.cpp

```diff
@@ -108,16 +122,42 @@ IECore::ObjectPtr readPointInstancer( pxr::UsdGeomPointInstancer &pointInstancer
 
 	// Prototype paths
 
+	const static bool gafferUSDPointInstancersRelativeProtoypes = checkEnvFlag( "GAFFERUSD_POINTINSTANCER_RELATIVEPROTOTYPES", true );
+	const static bool gafferSceneInstancerExplicitAbsolutePaths = checkEnvFlag( "GAFFERSCENE_INSTANCER_EXPLICITABSOLUTEPATHS", false );
+
 	pxr::SdfPathVector targets;
 	Canceller::check( canceller );
 	pointInstancer.GetPrototypesRel().GetForwardedTargets( &targets );
 
+	const pxr::SdfPath &primPath = pointInstancer.GetPath();
+
 	IECore::StringVectorDataPtr prototypeRootsData = new IECore::StringVectorData();
 	auto &prototypeRoots = prototypeRootsData->writable();
 	prototypeRoots.reserve( targets.size() );
 	for( const auto &t : targets )
 	{
-		prototypeRoots.push_back( t.GetString() );
+		if( !gafferUSDPointInstancersRelativeProtoypes )
+		{
+			prototypeRoots.push_back( t.GetString() );
+		}
+		else
+		{
+			if( t.HasPrefix( primPath ) )
+			{
+				if( gafferSceneInstancerExplicitAbsolutePaths )
+				{
+					prototypeRoots.push_back( t.MakeRelativePath( primPath ).GetString() );
```

Comment:
I think I'd be inclined to always use the `./` form for now, and just switch over to the implicit form unconditionally when we know that is supported by Gaffer.